dir.traversal.targets

```diff
@@ -28,33 +28,29 @@
   </Target>
 
   <!-- build vertical requires the FindBestConfiguration task which is currently in CoreFx.Tools.dll -->
-  <Import Project="buildvertical.targets" Condition="Exists('$(CoreFxToolsTaskDir)CoreFx.Tools.dll')" />
+<!-->  <Import Project="buildvertical.targets" Condition="Exists('$(CoreFxToolsTaskDir)CoreFx.Tools.dll')" /> -->
 
   <Target Name="BuildAllProjects">
     <PropertyGroup>
       <DefaultBuildAllTarget Condition="'$(DefaultBuildAllTarget)'==''">$(MSBuildProjectDefaultTargets)</DefaultBuildAllTarget>
     </PropertyGroup>
 
-    <MSBuild Targets="AnnotateProjectsWithConfiguration"
```

Comment:
I think that we'll need some of this in place to pass CI on non-Windows platforms.  I'm looking into it now.